content/nginx-one/k8s/add-ngf-helm.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Connect NGINX Gateway Fabric with Helm
+toc: true
+weight: 300
+nd-content-type: how-to
+nd-product: NGINX One
+---
+
+This document explains how to connect F5 NGINX Gateway Fabric to F5 NGINX One Console with Helm.
+Connecting NGINX Gateway Fabric to NGINX One Console enables centralized monitoring of all controller instances.
+
+Once connected, you'll see a **read-only** configuration of NGINX Gateway Fabric. For each instance, you can review:
+
+- Read-only configuration file
+- Unmanaged SSL/TLS certificates for Control Planes
+
+## Before you begin
```

Comment:
If the manifests page is going to be a direct copy of the existing manifest installation page, then the helm page might as well do the same and similarly use includes to prevent duplication.

Reply:
see [content/includes/ngf/installation/deploy-ngf-manifests.md](https://github.com/nginx/documentation/pull/902/files/154e935c76611d7e155200b54b1c9249c78ee8c2#diff-bfdd9cff33b17c650b9508d53c9388b086e834c5932cfa857e6e660e667b6ed9)